analyze/aps-dnc.c

```diff
@@ -2573,6 +2629,8 @@ void print_instance(INSTANCE *i, FILE *stream) {
 
 void print_edge_helper(DEPENDENCY kind, CONDITION *cond, FILE* stream) {
   if (stream == 0) stream = stdout;
+  if ((kind & DEPENDENCY_MAYBE_SIMPLE) == DEPENDENCY_MAYBE_SIMPLE) fputc('s', stream);
```

Comment:
To print circular information, I'd recommend first checking if MAYBE_SIMPLE is *missing* in which case print something like a 'o' (to mean circular/monotone) and then set the flag so that the next switch will print better.